src/iuse_actor.cpp

```diff
@@ -439,6 +451,10 @@ void iuse_transform::finalize( const itype_id & )
 
 void iuse_transform::info( const item &it, std::vector<iteminfo> &dump ) const
 {
+
+    if( !target_group.is_empty() ) {
+        return;
+    }
```

Comment:
That should generate some informative output at least, so you know you can transform it when you look at iteminfo. Ideally a proper description of the item group, but that might be extremely complicated with nested distribution groups, so at least in my opinion some generic sentence would be enough for now.

Reply:
I can't even refer to the result output, so I'm not sure what to write here. "Activation can transform into.. Something?"

Reply:
"Into several different items"?